packages/app/src/app/pages/Sandbox/Editor/Workspace/items/Live/LiveInfo/LiveInfo.tsx

```diff
@@ -0,0 +1,338 @@
+                              onClick={() => onFollow({ liveUserId: null })}
+                            />
+                          </Tooltip>
+                        ) : (
+                          <Tooltip content="Follow along">
+                            <FollowIcon
+                              onClick={() => onFollow({ liveUserId: user.id })}
+                            />
+                          </Tooltip>
+                        )}
+                      </IconContainer>
+                    )}
+
+                    {isOwner && mode === 'classroom' && (
+                      <IconContainer style={{ marginLeft: '0.25rem' }}>
+                        <Tooltip content={'Make spectator'}>
+                          <RemoveIcon
+                            onClick={() =>
+                              onRemoveEditorClicked({ liveUserId: user.id })
+                            }
+                          />
+                        </Tooltip>
+                      </IconContainer>
+                    )}
+                  </>
+                }
+                type="Editor"
+                user={user}
+                users={users}
+              />
+            ))}
+          </Users>
+        </Margin>
+      )}
+
+      <Margin top={1}>
+        <SubTitle>Users</SubTitle>
+
+        <Users>
+          {otherUsers.length ? (
+            otherUsers.map(user => (
+              <User
+                currentUserId={liveUserId}
+                key={user.id}
+                sideView={
+                  <>
+                    {mode !== 'classroom' && user.id !== liveUserId && (
+                      <IconContainer>
+                        {followingUserId === user.id ? (
+                          <Tooltip content="Stop following">
+                            <UnFollowIcon
+                              onClick={() => onFollow({ liveUserId: null })}
+                            />
+                          </Tooltip>
+                        ) : (
+                          <Tooltip content="Follow along">
+                            <FollowIcon
+                              onClick={() => onFollow({ liveUserId: user.id })}
+                            />
+                          </Tooltip>
+                        )}
+                      </IconContainer>
+                    )}
+
+                    {isOwner && mode === 'classroom' && (
+                      <IconContainer style={{ marginLeft: '0.25rem' }}>
+                        <Tooltip content={'Make editor'}>
+                          <AddIcon
+                            onClick={() =>
+                              onAddEditorClicked({ liveUserId: user.id })
+                            }
+                          />
+                        </Tooltip>
+                      </IconContainer>
+                    )}
+                  </>
+                }
+                type="Spectator"
+                user={user}
+                users={users}
```

Comment:
This component is huge, would you mind splitting it up?

Reply:
Split this one up 🙂